src/solana.ts

```diff
@@ -31,3 +32,9 @@ export type SolanaChain = typeof SOLANA_CHAINS[number];
 export function isSolanaChain(chain: IdentifierString): chain is SolanaChain {
     return SOLANA_CHAINS.includes(chain as SolanaChain);
 }
+
+export function isVersionedTransaction(
```

Comment:
This is from https://github.com/solana-labs/wallet-adapter/blob/55ff7e0a7c68a3ed12771f0feb06331a36b9509c/packages/core/base/src/transaction.ts#L9